.circleci/config.yml

```diff
@@ -97,12 +99,13 @@ jobs:
       - run:
           name: run tests
           command: |
-            TEST_ARGS="--backend=<<parameters.backend>> -v -s" DEV=n make savepoint_tests
+            TEST_ARGS="--backend=<<parameters.backend>> -v -s" DEV=y make savepoint_tests
           no_output_timeout: 3h
       - save_cache:
-          key: v1-gt_cache-serial-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
+          key: v3-gt_cache-serial-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
           paths:
             - .gt_cache
+            - .gt_cache_000000
```

Comment:
I thought `GT_CACHE_ROOT` would pipe the cache directory?

Reply:
We don't set GT_CACHE_ROOT for this plan, though?

Reply:
Right, looks like only for running 54 ranks mpi test